core/deployment/src/main/java/io/quarkus/artemis/core/deployment/ArtemisCoreConfiguredBuildItem.java

```diff
@@ -1,9 +0,0 @@
-package io.quarkus.artemis.core.deployment;
-
-import io.quarkus.builder.item.SimpleBuildItem;
-
-/**
- * Marker build item indicating that Artemis Core is configured
- */
-public final class ArtemisCoreConfiguredBuildItem extends SimpleBuildItem {
```

Comment:
Why remove it? It is a breaking change as some other libraries might depend on this to determine if they need to do extra work.

Reply:
done